$OpenBSD: patch-Makefile_PL,v 1.4 2016/04/01 19:32:48 sthen Exp $

comment-out this block, it uses an installed version of XML::SAX to write out
an ini file. this is provided by ${FILESDIR}/ParserDetails.ini instead.

--- Makefile.PL.orig	Sun Sep  4 22:37:48 2011
+++ Makefile.PL	Fri Apr  1 14:45:14 2016
@@ -21,6 +21,7 @@ sub MY::install {
 
     my $write_ini_ok = 0;
 
+=pod
     eval { require XML::SAX };
     if ($@) {
         $write_ini_ok = 1;
@@ -38,6 +39,7 @@ sub MY::install {
         }
     }
     
+=cut
     if ($write_ini_ok) {
         $script =~ s/install :: (.*)$/install :: $1 install_sax_pureperl/m;
         $script .= <<"INSTALL";
